script.js

```diff
@@ -0,0 +1,98 @@
+const Form = document.getElementById('booksform');
+const Bookscontainer = document.querySelector('.books');
+let myBooks = null;
+
+function makeGrey(book, index) {
+  const isGrey = book.node.classList.contains('grey');
+  if (index % 2 === 0 && !isGrey) {
+    book.node.classList.add('grey');
+  } else if (isGrey && index % 2 !== 0) {
+    book.node.classList.remove('grey');
+  }
+}
+
+const createBooksElements = (book) => {
+  const div = document.createElement('div');
+
+  const info = document.createElement('p');
+  info.innerText = `"${book.title}"\tby\t${book.name}`;
+
+  const remove = document.createElement('button');
+  remove.classList.contains('grey');
+  remove.innerHTML = 'Remove';
+  remove.addEventListener('click', () => {
+    myBooks.remove(book.id, div);
+  });
+
+  div.append(info, remove);
+  Bookscontainer.appendChild(div);
+
+  return div;
+};
+
+class Books {
+  constructor(array) {
+    if (array) {
+      this.array = array;
+    } else {
+      this.array = [];
+    }
+  }
+
+  get count() {
+    return this.array.length;
+  }
+
+    updateStorage = () => {
+      let string = '';
+      this.array.forEach((book, index) => {
+        book.id = index;
+        makeGrey(book, index);
+        string = `${string}${JSON.stringify(book)}|`;
+      });
+      string = string.substring(0, string.length - 1);
+      localStorage.setItem('books', string);
+    };
+
+    add(book) {
+      const node = createBooksElements(book);
+      book.node = node;
+      this.array.push(book);
+      this.updateStorage();
+    }
+
+    remove(i) {
+      Bookscontainer.removeChild(this.array[i].node);
+      this.array.splice(i, 1);
```

Comment:
- [**Optional**] Let's favor the use of array methods that don't mutate array like [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)